app/controllers/cloud_volume_backup_controller.rb

```diff
@@ -27,7 +27,7 @@ def volume_select
 
   def volume_form_choices
     assert_privileges("cloud_volume_backup_restore_to_volume")
-    volume_choices = ManageIQ::Providers::CloudManager::CloudVolume.all
+    volume_choices = CloudVolume.all
```

Comment:
I believe there are rules for where the backup can be restored. At least you will need to filter by the same EMS as backup. Maybe even the same tenant? @aufi can you advise?

Reply:
Sounds it didn't work according to those rules before. Well if anyone knows those rules @Hchrast can add them:)